sdk/storage/azure-storage-blob/downloadtofiledoesnotexist.txt

```diff
@@ -1 +0,0 @@
-default
```

Comment:
why is this file in there anyway?

Reply:
I think one of the unit tests create it but don't clean it up properly.